skills/dev/scripts/resolve_repository.py

```
@@ -0,0 +1,423 @@
+        result["reason_code"] = "origin_is_only_github_remote"
+        result["reason"] = (
+            f"`{remote_name}` resolves to {origin} and no other GitHub remote "
+            "or gh default disagrees"
+        )
+    return result
+
+
+def read_command(command: list[str], *, cwd: Path) -> tuple[int, str]:
+    """Run one read-only command and return its exit status and stdout."""
+    try:
+        completed = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)